src/txadmission.cpp

```diff
@@ -31,6 +32,15 @@
 
 using namespace std;
 
+static void TestConflictEnqueueTx(CTxInputData &txd);
+
+// The average commit batch size is used to limit the quantity of transactions that are moved from the defer queue
+// onto the inqueue.  Without this, if received transactions far outstrip processing capacity, transactions can be
+// shuffled between the in queue and the defer queue with little progress being made.
+const uint64_t minCommitBatchSize = 10000;
```

Comment:
nit: shouldn't we use all capital letter for `const`?

Reply:
no its a lowercase primitive